src/Components/People.js

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export default class People extends Component {
+  state = { people: [], input: "", name: "", age: "", gender: "" };
+
+  getPeople = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/people");
+      const peopleData = res.data;
+      this.setState({ people: peopleData });
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  componentDidMount = () => {
+    this.getPeople();
+  };
+
+  nameChecker = () => {
+    const { people, input } = this.state;
+    for (let i = 0; i < people.length; i++) {
+      if (input.toLowerCase() === people[i].name.toLowerCase()) {
```

Comment:
A better approach to this is using filter or find to just get hold of the one that matches